### PR TITLE
Store cache of Composer's cache directory

### PR DESCRIPTION
**Pull request by henriquemoody**
Related to #1606.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,7 +24,7 @@ before_install:
   - composer self-update
```
Comment (GrahamCampbell) [inline .travis.yml]: We don't need this self-update either. Can it be removed @whatthejeff?

Comment (henriquemoody) [inline .travis.yml]: Sorry, I've removed my fork because I thought it doesn't need changes, if we have to remove this I can send another pull request.

**Comment by whatthejeff** [comment]
Looks good to me. Thanks!

File: .travis.yml (hunk 1 above)
Comment (whatthejeff) [inline .travis.yml]: Yeah, let's just make a new PR and remove the self-update :)

Comment (whatthejeff) [inline .travis.yml]: Sorry for the inconvenience, @henriquemoody.

Comment (henriquemoody) [inline .travis.yml]: No problem.

Comment (henriquemoody) [inline .travis.yml]: #1609 is ready.

**Pull request closed without merging**